plugins/single_plugins/expo.cpp

```diff
@@ -22,12 +22,20 @@ class wayfire_expo : public wf::plugin_interface_t
 
     activator_callback toggle_cb = [=] (wf_activator_source, uint32_t)
     {
+        if (!output->activate_plugin(grab_interface))
```

Comment:
pretty sure `activate` does try to activate the plugin so just get the value from there.